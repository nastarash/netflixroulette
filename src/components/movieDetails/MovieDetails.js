import React from 'react';

class MovieDetails extends React.Component {
  constructor() {
    super();
    this.state = {};
  }

  render() {
    return (
      <div>
        <img src="" alt="bla-bla" />
        <h3>Tile</h3>
        <h4>Short Decrption</h4>
        <span>Duration</span>
        <span>Release Year</span>
        <span>Rating</span>
        <span>Description</span>
      </div>
    );
  }
}

export default MovieDetails;
