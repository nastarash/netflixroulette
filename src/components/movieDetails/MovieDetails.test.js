import React from 'react';
import renderer from 'react-test-renderer';
import MovieDetails from './MovieDetails';

it('render correctly', () => {
  const AppComponent = renderer.create(<MovieDetails />).toJSON();
  expect(AppComponent).toMatchSnapshot();
});
