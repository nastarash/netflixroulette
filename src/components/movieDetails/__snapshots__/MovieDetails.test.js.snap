// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div>
  <img
    alt="bla-bla"
    src=""
  />
  <h3>
    Tile
  </h3>
  <h4>
    Short Decrption
  </h4>
  <span>
    Duration
  </span>
  <span>
    Release Year
  </span>
  <span>
    Rating
  </span>
  <span>
    Description
  </span>
</div>
`;
